### ISSUE TITLE
Output pdf single/twoside

### ISSUE THREAD

**Issue by alterat** [italia/docs-italia-theme#108]
I pdf generati dai documenti di Docs Italia hanno tutti il layout `twoside` di Latex. 
Questo layout è ottimizzato per la stampa: una delle caratteristiche principali è la presenza di pagine bianche prima di ogni capitolo, che fanno in modo che l'inizio del capitolo capiti sempre su una pagina dispari. 

Da un lato questo potrebbe essere un bene, dall'altro potrebbe dare fastidio quando si cerca di leggere il pdf sullo schermo (in realtà questo è il comportamento che stiamo cercando di sradicare con Docs Italia...). 

Vorrei avere le vostre opinioni al riguardo, che ne pensate?

**Comment by pablopers** [comment]
Concordo con Alberto @atorin, in particolare penso che la presenza di pagine bianche (peraltro non sempre prima di ogni capitolo) potrebbe indurre a pensare ad un errore. Inoltre nell'ipotesi che si proceda al contrario (Alberto lo aveva ipotizzato per un documento che riguardava proprio la Città Metropolitana di Napoli), ossia, partire da una versione pubblicata su Docs Italia, generare da lì, il pdf e allegarlo all'atto di approvazione (firmato digitalmente) dal Sindaco, potrebbe essere un ostacolo.
